.github/workflows/ci.yaml

```diff
@@ -976,7 +976,7 @@ jobs:
       - changes
       - build-dylib
     if: github.ref == 'refs/heads/main' && needs.changes.outputs.docs-only == 'false' && !github.event.pull_request.head.repo.fork
-    runs-on: "ubuntu-22.04"
+    runs-on: ${{ github.repository_owner == 'coder' && 'depot-ubuntu-22.04-8' || 'ubuntu-latest' }}
```

Comment:
We are using depot 22.04 runner but ubuntu-latest will soon or have moved to 24.04 which may cause different behavior on forks vs orgs PRs. 
Suggestion to use `ubuntu-22.04`